README.md

```diff
@@ -416,7 +417,8 @@ given string with a given replacement string.
 Params:
 
 - `paths`: a list of files and/or directories in which to do the replacement.
-  May use template expressions (e.g. `{{.my_input}}`).
+  May use template expressions (e.g. `{{.my_input}}`). Directories will be
+  crawled recursively and every file underneath will be processed.
```

Comment:
What if you specify directories/files that overlap, will the same modification be applied multiple times?

Reply:
Let's leave that undocumented for now until #124 is fixed, since the behavior will shortly change, and it's an edge case.